R/fit.R

```diff
@@ -1549,3 +1549,56 @@ CmdStanDiagnose$set("public", name = "save_output_files", value = save_output_fi
 CmdStanDiagnose$set("public", name = "output_files", value = output_files)
 CmdStanDiagnose$set("public", name = "save_data_file", value = save_data_file)
 CmdStanDiagnose$set("public", name = "data_file", value = data_file)
+
+
+
+# as_draws ----------------------------------------------------------------
+#' Create a `draws` object from a CmdStanR fitted model object
+#'
+#' Create a `draws` object supported by the \pkg{posterior} package. These
+#' methods are just wrappers around CmdStanR's [`$draws()`][fit-method-draws]
+#' method provided for convenience.
+#'
+#' @aliases as_draws
```

Comment:
Do we mention that users can now do things like

```
model %>% as_draws_XXX()
```

or is that for a vignette with an example?

Reply:
In the `@examples` section of the doc I have this:  

```
#' # posterior's as_draws_*() methods will also work
#' posterior::as_draws_rvars(fit)
#' posterior::as_draws_list(fit)
```

Do you think that's sufficient?

Reply:
Yeah, should be good.